package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oav",
-  "version": "2.11.6",
+  "version": "2.11.7",
```

Comment:
2.11.7 isn't released yet, no need to bump, you could add changelog to 2.11.7 section.

You may resolve branch conflict later when this [PR](https://github.com/Azure/oav/pull/752) is merged